### DISCUSSION TITLE
Do I need to load Tailwind CSS?

### DISCUSSION THREAD

**Discussion by shinokada** [themesberg/flowbite#76] [Q&A]
I thought the Flowbite comes with Tailwind CSS.
But when I try to use [`basis-1/4`](https://tailwindcss.com/docs/flex-basis), it doesn't work.

I'm wondering if I need to load Tailwind CSS as well.

**Comment by zoltanszogyenyi** [comment]
Hey @shinokada,

Flowbite is a plugin for Tailwind CSS, so by using Flowbite you need to have a Tailwind CSS project.

Indeed, Flowbite's CDN will work because Tailwind CSS would be loaded already, but that is not the recommended way of working with it because the CSS wouldn't be purged. (resulting in large CSS files)
